src/Microsoft.Android.Sdk.ILLink/PreserveLists/System.Private.CoreLib.xml

```diff
@@ -3,5 +3,6 @@
 <linker>
 	<assembly fullname="System.Private.CoreLib">
 		<type fullname="System.Globalization.GlobalizationMode" />
+		<type fullname="System.Runtime.CompilerServices.InternalsVisibleToAttribute" />
```

Comment:
This fixed a crash in Mono.Android-Tests:

    Can't find custom attr constructor image: Mono.Android.dll mtoken: 0x0a000464 due to:
    Could not resolve type with token 010000fb from typeref (expected class 'System.Runtime.CompilerServices.InternalsVisibleToAttribute' in assembly 'System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e') assembly:System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e type:System.Runtime.CompilerServices.InternalsVisibleToAttribute member:(null)